### DISCUSSION TITLE
Exactly what means when a Tracker is n/a ?

### DISCUSSION THREAD

**Discussion by dantearmok** [transmission/transmission#5440] [Support]
n/a, to me, means not applicable.  But that doesn't really make sense in the Inspector window.

Right now, I've got a private-site torrent that says:
![Screen Shot 2023-04-23 at 1 49 24 AM](https://user-images.githubusercontent.com/20881295/233822509-180d5c15-f61f-4929-8698-d4407d0180e1.jpg)

What exactly does all those "N/A"'s mean?

And one step further -- The other Inspector pane says "Private Torrent, non-tracker peer discovery disabled". Now, presumably that means that the only way to leech/seed the torrent is by first talking to the tracker. But... this torrent is leeching quickly! 10+ Mbps.  But the tracker says N/A. ??????

*edit*
I have four torrents from that private-site running, and Transmission is spiking above 93.6 Mbps.  Nice.

Thx.

**Comment by maxz** [comment]
Not available. And in this case it just means that the particular information is not public.
